mods/ts/sequences/infantry.yaml

```diff
@@ -530,7 +529,6 @@ umagon:
 	die6: electro
 		Length: *
 		Tick: 80
-	icon: umagicon
```

Comment:
That is not a valid point. It is there and has to be assigned somewhere so we keep track of it.